src/ray/raylet/node_manager.cc

```diff
@@ -1480,8 +1479,10 @@ void NodeManager::HandleUpdateResourceUsage(
     rpc::SendReplyCallback send_reply_callback) {
   rpc::ResourceUsageBroadcastData resource_usage_batch;
   resource_usage_batch.ParseFromString(request.serialized_resource_usage_batch());
-
-  if (resource_usage_batch.seq_no() != next_resource_seq_no_) {
+  // When next_resource_seq_no_ == 0 it means it just started.
+  // TODO: Fetch a snapshot from gcs for lightweight resource broadcasting
+  if (next_resource_seq_no_ != 0 &&
+      resource_usage_batch.seq_no() != next_resource_seq_no_) {
```

Comment:
Handle `resource_usage_batch.seq_no() > next_resource_seq_no_` and `resource_usage_batch.seq_no() < next_resource_seq_no_` separately?

Reply:
Sounds good. I previously wanted to handle this one after we have lightweight resource broadcasting. But I guess here probably just discard the stale seq_no